src/main/java/frc/robot/subsystems/InterfaceSubsystem.java

```diff
@@ -35,20 +42,240 @@ public InterfaceSubsystem(
    *
    * <p>Example: Driver holding "A" button, robot auto-navigates to selected pole
    */
+
+  /**
+   * Drives to the selected location Works by converting Pose2d of the branch selected to a
+   * transform then pathfinder-ing to it.
+   */
   public void driveToLoc(InterfaceExecuteMode loc) {
     switch (loc) {
       case REEF:
         switch (pole) {
           case "a":
-            // Drive to pole A and align once nearby
-            // TODO: Add the rest of the branch positions to align to
+            targetTransform =
+                new Transform2d(
+                    new Translation2d(
+                        FieldConstants.getSetPoint(Place.A_TREE).getX(),
+                        FieldConstants.getSetPoint(Place.A_TREE).getY()),
+                    FieldConstants.getSetPoint(Place.A_TREE).getRotation());
+
+            DriveCommands.goToTransformWithPathFinder(drive, targetTransform)
+                .andThen(DriveCommands.goToTransform(drive, targetTransform))
+                .beforeStarting(
+                    () -> {
+                      DriveCommands.goToTransform(drive, targetTransform).cancel();
+                      DriveCommands.goToTransformWithPathFinder(drive, targetTransform).cancel();
+                    });
```

Comment:
🎨 This is repeated every time. Can you do the work in the switch to set `targetTransform` and then do this once?